Flank/flank
# fix: Do not upload/read files when path starts with gs://

Fixes #1971 

When user provides `gs://` path in `other-files` we should not upload it again.
```
Files.readAllBytes(Paths.get(file))
```
will never succeed when the path starts with `gs://` 

## Test Plan
> How do we know the code works?

1. [flank test bucket](https://console.cloud.google.com/storage/browser/flank-open-source.appspot.com/test?project=flank-open-source&pageState=(%22StorageObjectListTable%22:(%22f%22:%22%255B%255D%22))&prefix=&forceOnObjectsSortingFiltering=false)
1. run `./gradlew flankFullRun`
1. prepare config with:
    ```
    other-files:
        /sdcard/[some file name]: gs://[path to file uploaded to the bucket, you can reuse existing one]
    ```
1. run tests
1. go to result bucket and check if there is a file from config under `/artifacts/sdcard/` path
